### ISSUE TITLE
Why are Facodec and Ns3_facodec different?

### ISSUE THREAD

**Issue by ndhuynh02** [open-mmlab/Amphion#342]
I am looking at that Model code of 2 folders `facodec` and `ns3_facodec`. I know that `ns3_facodec` is the training code for Facodec. However, I am witnessing some differences between 2 architecture:
- First of all, there are no LSTMs in the official Facodec in both Encoder and Decoder
- Secondly, the timbre encoder is kinda different. Even though both are using Transformer, I am seeing that they are not the same.
- The generator loss is the combination of multiple losses by some weights. But as I look at the NaturalSpeech3 paper at the Appendix part, it is clearly that the weights are not like in the paper, rather than the DAC paper
